public/New-DbaDbUser.ps1

```diff
@@ -88,136 +91,127 @@ function New-DbaDbUser {
 
         Creates a new sql user named 'claudio@********.onmicrosoft.com' mapped to Azure Active Directory (AAD) in the specified database.
 
+    .EXAMPLE
+        PS C:\> New-DbaDbUser -SqlInstance sqlserver2014 -Database DB1 -Username user1 -Password (ConvertTo-SecureString -String "DBATools" -AsPlainText)
+
+        Creates a new cointained sql user named user1 in the database given database with the password specified.
     #>
     [CmdletBinding(SupportsShouldProcess, ConfirmImpact = "Medium")]
     param(
-        [parameter(Mandatory, Position = 1)]
+        [parameter(Mandatory = $True, Position = 1)]
         [DbaInstanceParameter[]]$SqlInstance,
         [PSCredential]$SqlCredential,
         [string[]]$Database,
         [string[]]$ExcludeDatabase,
-        [switch]$IncludeSystem,
+        [switch]$IncludeSystem = $False,
+        [Parameter(ParameterSetName = "NoLogin", Mandatory = $False)]
+        [Parameter(ParameterSetName = "Login", Mandatory = $True)]
         [string]$Login,
-        [string]$Username = $Login,
+        [Parameter(ParameterSetName = "Login", Mandatory = $False)]
+        [Parameter(ParameterSetName = "NoLogin", Mandatory = $True)]
+        [Parameter(ParameterSetName = "ContainedSQLUser", Mandatory = $True)]
+        [Parameter(ParameterSetName = "ContainedAADUser", Mandatory = $True)]
+        [string]$UserName,
```

Comment:
Why remove the default value? This is a big breaking change that should not be required.